### PR TITLE
2019/kw15/design_differences_post

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## Pullrequest

image

fixed design differences between alpha & nitro - Post Page
- [X] Title and Author Position reversed
- [X] Design & Format of Post-Date + Position
- [X] No Category Names
- [X] Actions/Reactions & Tags reversed

### Issues

- [X] fixes https://github.com/Human-Connection/Human-Connection/issues/392

### Checklist

- [X] None

### How2Test

- [X] None

### Todo

- [X] None

### CODE REVIEW

**Comment by roschaefer** [comment]
@ulfgebhardt I think the build server is failing for good reason, could you check?

**Comment by Tirokk** [comment]
Moin @roschaefer , the Jest component test of `RelativeDateTime` works fine and looks nice. Congrats !!!

File: webapp/components/RelativeDateTime/index.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import formatRelative from 'date-fns/formatRelative'
+import { enUS, de, nl, fr, pt, es /*, pl*/ } from 'date-fns/locale'
+const locales = {
+  en: enUS,
+  de,
+  nl,
+  fr,
+  es,
+  pt
+  //pl
```
Comment (roschaefer) [inline webapp/components/RelativeDateTime/index.vue]: What's wrong with Polish? :thinking: 

@ulfgebhardt by the way, I think I missed one test case: What happens, if you have a `i18n.locale()` but this locale is not in the list above? Could you create a test case for it? I would like to have a default in that case.

**Comment by roschaefer** [comment]
@ulfgebhardt the build server fails at `✖ 06.WritePost.feature` on both builds. I don't think this is a false positive (test fails, but implementation is fine) but a true positive (test fails because implementation is broken). Could you run the cypress tests locally and have a check?

File: webapp/components/RelativeDateTime/index.vue (hunk 1 above)
Comment (ulfgebhardt) [reply webapp/components/RelativeDateTime/index.vue]: Polish is currently not implemented on library side.

**Comment by ulfgebhardt** [comment]
image
image
image
image

Cypress tests are not working with chrome or electron
```
yarn cypress:setup
yarn cypress:run (changed browser)
```

Error message from travis is not reproducable
image
image

Posts can be created - works as expected

**Pull request merged**